pkg/cmd/server/origin/asset.go

```diff
@@ -220,8 +233,14 @@ func (c *AssetConfig) addHandlers(mux *http.ServeMux) error {
 		KubernetesVersion: kVersionInfo.GitVersion,
 		OpenShiftVersion:  oVersionInfo.GitVersion,
 	}
+
+	extensionPropertyKeys, extensionPropertyValues := extensionPropertyArrays(c.Options.ExtensionProperties)
+	extensionProps := assets.WebConsoleExtensionProperties{
```

Comment:
an array of WebConsoleExtensionProperty structs with Key/Value fields might be better than parallel arrays